records_mover/db/vertica/loader.py

```diff
@@ -66,6 +70,10 @@ def load_from_fileobj(self,
             if rawconn is not None:
                 rawconn.close()
 
+    def load_failure_exception(self) -> Type[Exception]:
+        return vertica_python.errors.CopyRejected
+
+    # TODO: Can't this be part of something inherited?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker